### PR TITLE
fix(missing date pick automation locator)

### PR DESCRIPTION
**Pull request by TiManGames**
Fixed testId attribute  name

### CODE REVIEW

**Review by ItayBenMoshe — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx [hunk 1]
```diff
@@ -125,7 +125,7 @@ const InteractionEventForm: React.FC<Props> = (
                     control={methods.control}
                     render={(props) => (
                       <TimePick
-                        test-id='contactLocationStartTime'
+                        testId={'contactLocationStartTime'}
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx:128]: you dont need {}

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx [hunk 2]
```diff
@@ -144,7 +144,7 @@ const InteractionEventForm: React.FC<Props> = (
                     control={methods.control}
                     render={(props) => (
                       <TimePick
-                        test-id='contactLocationEndTime'
+                        testId={'contactLocationEndTime'}
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx:147]: also here

Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx]: Fixed.

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx (hunk 1 above)
Comment (TiManGames) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx]: Fixed.

**Review by ItayBenMoshe — APPROVED** [review]

**Review by OrelKaravani — APPROVED** [review]

**Pull request merged**